yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -454,10 +478,16 @@ private[spark] class Client(
      *   (3) whether to add these resources to the classpath
      */
     val cachedSecondaryJarLinks = ListBuffer.empty[String]
+    val files = sparkConf.get(FILES_TO_DISTRIBUTE).map(p => Utils.resolveURIs(p))
+      .orElse(sys.env.get("SPARK_YARN_DIST_FILES"))
```

Comment:
that would be ok, but I think you have already removed most of them, it looks like there are only a couple left in this file so it might be just as easy to do here.